resources/api/reservation.py

```diff
@@ -115,6 +116,15 @@ def __init__(self, *args, **kwargs):
             for field_name in required:
                 self.fields[field_name].required = True
 
+    def get_extra_fields(self, includes, context):
+        from .resource import ResourceSerializer
+
+        """ Define extra fields that can be included via query parameters. Method from ExtraDataMixin."""
+        extra_fields = {}
+        if 'resource' in includes:
```

Comment:
`ResourceSerializer` has used `unit_detail` as the key to get the full unit data included, use the same notation here.

Reply:
changed the keyword